db/seeds/default_video.rb

```diff
@@ -0,0 +1,8 @@
+video1 = Video.create(:description => 'math, learn it', :title => 'Calculus AB', :youtube_id => 'bHNczNvOnGc')
+video2 = Video.create(:description => 'this is an asian film', :title => 'Asia and a Motorbike', :youtube_id => 'yJRy9o8gv6c')
+video3 = Video.create(:description => 'darren and naseem and brendan in bechtel', :title => 'You know it', :youtube_id => '6joOVjEemh4')
+video4 = Video.create(:description => 'potato', :title => 'Again and a Again', :youtube_id => '6joOVjEemh4')
+video5 = Video.create(:description => 'blah', :title => 'Again', :youtube_id => '6joOVjEemh4')
+
+
```

Comment:
Unnecessary extra line endings. Keep one so Git doesn't complain; the reset should go.